test/openAPI/features/support/helpers/helpers.js

```diff
@@ -13,6 +13,13 @@ module.exports = {
   databaseDeleteResponseSchema: { type: 'string' },
   databaseReadEndpoint: 'database/{id}',
   databasesEndpoint: 'databases',
+  dataCreateEndpoint: 'data/{registryname}/{versionnumber}/create',
+  dataCreateRequestBody: {
```

Comment:
I'd move the creation data to the feature file and invoke it in Examples table few times.